packages/fe-container/src/components/Composer.tsx

```diff
@@ -6,20 +6,23 @@ import viewEngine from '../composer/ViewEngine'
 import useConfiguration from '../hooks/useConfiguration'
 
 const propTypes = {
-  configurationName: PropTypes.string.isRequired
+  configurationName: PropTypes.string.isRequired,
+  currentUser: PropTypes.object
```

Comment:
Can you define the shape of the object?

Reply:
see 2 issues below. i'd rather keep currentUser general. I don't need its shape in microlc-element-composer.

Reply:
You need its shape, because you must validate the kind of object that you inject to web components: if the shape change, you could break the underlying implementation.